src/views/Range/RangeChart.tsx

```diff
@@ -188,9 +191,9 @@ const RangeChart = (props: {
             type="number"
             domain={[
               (dataMin: number) =>
-                Math.min(dataMin, askPrice, bidPrice, parseBigNumber(rangeData.wall.low.price, 18)) * 0.95,
+                Math.min(dataMin, askPrice, bidPrice, parseBigNumber(rangeData.low.wall.price, 18)) * 0.95,
```

Comment:
What is 0.95 and 1.05 referring to?

Reply:
it's just a scaling thing for the charts. it adds some padding to the top and bottom so we're not pegging the walls to the very top and bottom of the chart